File: frontend/src/type/filelibrary/FileLibraryStatisticsVo.ts
/**
 * 文件库统计 VO
 */
export interface FileLibraryStatisticsVo {
    /**
     * 文件数量
     */
    fileCount: number;

    /**
     * 存储大小（字节）
     */
    storageUsed: number;
}

File: backend/src/main/java/cn/yang/nebula/agent/business/file/library/vo/FileLibraryStatisticsVo.java
package cn.yang.nebula.agent.business.file.library.vo;

import lombok.Builder;
import lombok.Data;

/**
 * 文件库统计 VO
 *
 * @author 未见清海
 */
@Data
@Builder
public class FileLibraryStatisticsVo {

    /**
     * 文件数量
     */
    private Integer fileCount;

    /**
     * 存储大小（字节）
     */
    private Long storageUsed;
}
